planning/planners/src/encode.rs

```diff
@@ -387,9 +387,6 @@ fn enforce_refinement(t: TaskRef, supporters: Vec<TaskRef>, model: &mut Model) {
 
     // if a supporter is present, then all its parameters are unified with the ones of the supported task
     for s in &supporters {
-        // if the supporter is present, the supported is as well
-        assert!(model.state.implies(s.presence, t.presence));
```

Comment:
Instead of removing it, you should enforce that your encoding respect this. It will likely help a lot for performance as well.

Reply:
It seems that it was already removed, but was left there after the last rebase. It was a mistake.
However, if I understand well is has been moved in the refinement constraint: now the refinement constraints have both the scopes of the task and the supporters.

Reply:
Ok, indeed it must have been removed when allowing refinement of multiple tasks.